### ISSUE TITLE
Module gets "404: Not Found" while trying to send a message immediately after Webhook object creation

### ISSUE THREAD

**Issue by liddack** [JoeBanks13/webhook-discord#8]
Whenever I try to send a message using `Hook.info()` (or whichever method) right after creating a new Webhook object, as in:

```javascript
var Webhook = require("webhook-discord")
var Hook = new Webhook("WEBHOOK_URL")
Hook.err('Test','test')
```
I get this error: 

```
(node:11404) UnhandledPromiseRejectionWarning: {"code": 0, "message": "404: Not Found"}
(node:11404) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). (rejection id: 2)
(node:11404) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
```
After debugging the module code, I found out `this.endpoint` was not correctly formed at the moment of `Hook.err()` call, since the module relies on a `Promise` to fetch webhook URL properties (like `id` and `token`). As a `console.log()` shows:

```javascript
...
sendPayload(payload) {
        console.log(this.endpoint);
        // logs "https://discordapp.com/api/v6/webhooks///slack"
        return new Promise((resolve, reject) => {
            snekfetch.post(this.endpoint)
                .send(payload)
...
```

I already made a little fix/workaround myself, but I'm not confident enough to send a pull request 😅 so I created an issue instead.

**Gist with my workaround**: https://gist.github.com/liddack/758e14c7161d814e74fa0a49c05ab585

**Comment by JoeBanks13** [comment]
Hey! This is a known issue but I just never got around to addressing a fix. I'll try get around to implementing your fix (which is greatly appreciated) into the library, however if you want feel free to open a pull request so I can look over and give it more of a test. Will leave this open for now :)

**Comment by JoeBanks13** [comment]
This is fixed in version 2.2.2, see [this commit](https://github.com/JoeBanks13/webhook-discord/commit/ee13d5fb3b942bdf63e3eba6e7211c85b658ec79)
